lib/demos/cupertino/cupertino_alert_demo.dart

```diff
@@ -50,118 +105,120 @@ class _CupertinoAlertDemoState extends State<CupertinoAlertDemo> {
     return '';
   }
 
-  void _showDemoDialog({BuildContext context, Widget child}) {
-    showCupertinoDialog<String>(
+  static Route<String> _alertDemoDialog(
+      BuildContext context, Object arguments) {
```

Comment:
nit (although I am not super familiar with the code formatting style guide here): I'd add a trailing comma behind `arguments` and format this in multiple lines.